semgrep-core/src/engine/Match_tainting_mode.ml

```diff
@@ -420,22 +443,8 @@ let check_fundef lang fun_env taint_config opt_ent fdef =
   check_stmt ?name ~in_env lang fun_env taint_config
     (H.funcbody_to_stmt fdef.G.fbody)
 
-(* TODO: Pass a hashtable to cache the CFG of each def, otherwise we are
```

Comment:
why we don't need this code anymore?

Reply:
When adding the taint trace to DeepSemgrep, this was a hack that I used to keep track of the taint `finding` associated with the reported pattern match. But now the pattern match already carries the taint trace.